examples/defaulter-gen/output_tests/wholepkg/vendor/k8s.io/apimachinery/pkg/runtime/mock.go

```diff
@@ -0,0 +1,22 @@
+/*
```

Comment:
Why do you want the generated file to be buildable? is the test only making sure that the output is what we expecting?